friendface-web/src/main/java/com/scottlogic/grad_training/friendface/SpringFoxConfig.java

```diff
@@ -0,0 +1,75 @@
+package com.scottlogic.grad_training.friendface;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpMethod;
+import springfox.documentation.builders.*;
+import springfox.documentation.service.*;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spi.service.contexts.SecurityContext;
+import springfox.documentation.spring.data.rest.configuration.SpringDataRestConfiguration;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger.web.SecurityConfiguration;
+import springfox.documentation.swagger.web.SecurityConfigurationBuilder;
+
+import java.util.Arrays;
+
+@Configuration
+@Import(SpringDataRestConfiguration.class)
+public class SpringFoxConfig {
+
+    private final String CLIENT_ID = "root";
+    private final String CLIENT_SECRET = "Nx2z2HW4x";
```

Comment:
In production code this would be really bad - you have your secret exposed in code.
Ways around this would be 
- Storing an encrypted secret that can only be decrypted by a private key that is not exposed
- Getting the value from system properties
- Asymmetric encryption to generate a keystore (industry standard)